source/renderer/app/components/staking/delegation-setup-wizard/DelegationStepsChooseWalletDialog.js

```diff
@@ -0,0 +1,178 @@
+// @flow
+import React, { Component } from 'react';
+import { defineMessages, intlShape, FormattedHTMLMessage } from 'react-intl';
+import { Select } from 'react-polymorph/lib/components/Select';
+import { SelectSkin } from 'react-polymorph/lib/skins/simple/SelectSkin';
+import { Stepper } from 'react-polymorph/lib/components/Stepper';
+import { StepperSkin } from 'react-polymorph/lib/skins/simple/StepperSkin';
+import styles from './DelegationStepsChooseWalletDialog.scss';
+import DialogCloseButton from '../../widgets/DialogCloseButton';
+import DialogBackButton from '../../widgets/DialogBackButton';
+import Dialog from '../../widgets/Dialog';
+
+const messages = defineMessages({
+  title: {
+    id: 'delegation.setup.steps.dialog.title',
+    defaultMessage: '!!!Delegation Setup',
+    description:
+      'Title "Delegation Setup" on the delegation setup "choose wallet" step dialog.',
+  },
+  description: {
+    id: 'delegation.setup.chooseWallet.step.dialog.description',
+    defaultMessage:
+      '!!!Choose a wallet with funds you would like to delegate to a stake pool. Selected wallet needs to have a minimum of <span>1 ada</span>.',
```

Comment:
Better not to leave 1 hardcoded, as it might change, but rather as a variable. You're already using `<FormattedHTMLMessage {...messages.description} />` so it's just a matter of passing the 1 as variable.